src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -1948,6 +1948,7 @@ namespace Microsoft.FSharp.Collections
         interface IEnumerable
         interface IReadOnlyCollection<'T>
         interface IReadOnlyList<'T>
+        interface IComparable
```

Comment:
At the moment List implements a whole bunch of interfaces, not just IComparable:

![image](https://user-images.githubusercontent.com/4236651/81003232-cca04100-8e4a-11ea-889d-8d6d37569fd9.png)